### PR TITLE
fix spelling and add app version to error message

### PR DESCRIPTION
**Pull request by aprooks**

### CODE REVIEW

File: src/Etg.Yams.Core/Deploy/RemoteApplicationDeploymentDirectory.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public RemoteApplicationDeploymentDirectory(IDeploymentRepository deploymentRepo
                     {
                         if (!await _deploymentRepository.HasApplicationBinaries(appIdentity))
                         {
-                            Trace.TraceError($"Could not binaries for application {appIdentity} in the yams repository");
+                            Trace.TraceError($"Could not find binaries for application {appIdentity} of {version} in the yams repository");
```
Comment (nehmebilal) [inline src/Etg.Yams.Core/Deploy/RemoteApplicationDeploymentDirectory.cs]: AppIdentity already contains the version and ToString prints both the id and the version.

Comment (aprooks) [inline src/Etg.Yams.Core/Deploy/RemoteApplicationDeploymentDirectory.cs]: yep, it does. Removing version

**Comment by aprooks** [comment]
Do I need to rebase commits into one?

**Comment by nehmebilal** [comment]
Would be cleaner :) You can just reset to HEAD~2 then commit again. You'll have to push the branch with -f.

```
git reset HEAD~2
git commit -m"Fix spelling in error message"
git push aprooks patch-1 -f
```

**Comment by nehmebilal** [comment]
I just used the "squash and merge" option of GitHub. Thanks for the fix!

**Pull request merged**